src/AutoRecordings.cpp

```diff
@@ -172,22 +172,22 @@ PVR_ERROR AutoRecordings::SendAutorecAdd(const PVR_TIMER &timer)
   if (strcmp(timer.strDirectory, "/") != 0)
     htsmsg_add_str(m, "directory", timer.strDirectory);
 
-  /* Note: not sending any of the following for values will be interpreted by tvh as "any". */
+  /* Note: not sending any of the following three values will be interpreted by tvh as "any". */
   if (timer.iClientChannelUid >= 0)
     htsmsg_add_u32(m, "channelId", timer.iClientChannelUid);
 
   if (timer.startTime > 0)
   {
-    /* approx start time in minutes from midnight. tvh will match value +- 15 mins. */
+    /* Exact start time (minutes from midnight). */
     struct tm *tm_start = localtime(&timer.startTime);
-    htsmsg_add_s32(m, "approxTime", tm_start->tm_hour * 60 + tm_start->tm_min);
+    htsmsg_add_s32(m, "start", tm_start->tm_hour * 60 + tm_start->tm_min);
   }
 
-  if ((timer.endTime - timer.startTime) > 0)
+  if (timer.endTime > 0)
   {
-    /* min/max duration: endtime -+ 15 min */
-    htsmsg_add_u32(m, "minDuration", timer.endTime - timer.startTime - 15 * 60);
-    htsmsg_add_u32(m, "maxDuration", timer.endTime - timer.startTime + 15 * 60);
+    /* Exact stop time (minutes from midnight). */
+    struct tm *tm_stop = localtime(&timer.endTime);
+    htsmsg_add_s32(m, "startWindow", tm_stop->tm_hour * 60 + tm_stop->tm_min);
```

Comment:
I bit late but this seems messing thing up...

End time should reflect to endWindow instead of startWindow, as we don't have an endWindow we should use maxduration for this..